lambda-code/reliability/lib/templates.ts

```diff
@@ -12,7 +12,13 @@ export type TemplateInfo = {
 
 export async function getTemplateInfo(formID: string): Promise<TemplateInfo | null> {
   try {
-    const templates = await DatabaseConnectorClient<
+    const postgresConnector =
+      await PostgresConnector.defaultUsingPostgresConnectionUrlFromAwsSecret(
+        process.env.DB_URL ?? "",
+        process.env.LOCALSTACK === "true"
```

Comment:
As long as the value is defined or falsey.  Allows devs to use True, true, 1, etc.  
```suggestion
        Boolean(process.env.LOCALSTACK)
```